### ISSUE TITLE
Autogeneration of a user account on NeuroHub after ORCID Auth success

### ISSUE THREAD

**Issue by dlq** [neurohub/neurohub_asana#101]
Upon successful Authorization from ORCID, if the email address is not associated to an existing user then create a user.

**Comment by dlq** [comment]
➤ Xavier Lecours commented:

Adding provisory dates. This is not a priority for MVP

**Comment by prioux** [comment]
Can be done, it's not too difficult, but I dont' like it much because it allows anyone in the world to create a neurohub account, and the DB will soon be overwhelmed with accounts that are used only once by curious people and then abandoned.

I think we might consider doing this only for email addresses that are in some specific domains: mcin.ca, mcgill.ca, etc
